lib/tools/system-calls.js

```diff
@@ -208,7 +208,15 @@ systemCallMethods.getBinaryFromPath = async function getBinaryFromPath (binaryNa
  * @property {string} udid - The device udid.
  * @property {string} state - Current device state, as it is visible in
  *                            _adb devices -l_ output.
- * Will have additional properties if the `long` flag was set.
+ */
+
+/**
+ * @typedef {Device} VerboseDevice Additional properties returned when `verbose` is true.
+ * @property {string} product
```

Comment:
Could we also add a more detailed property description and maybe one simple example?